webapp/lib/view.dart

```diff
@@ -461,6 +467,28 @@ class ContentView {
     if (urlFilters['project'] != null) {
       projectSelectorView.setActiveProject(urlFilters['project']);
     }
+
+    if (urlFilters['period-filter'] != null) {
+      var period;
+      switch (urlFilters['period-filter']) {
+        case 'alltime':
+          period = 'ChartPeriodFilters.alltime';
+          break;
+        case 'days1':
+          period = 'ChartPeriodFilters.days1';
+        break;
```

Comment:
nit: `break` should have the same indent as the line above it. Same for below.

Reply:
Fixed that.